### ISSUE TITLE
Add locale online service to translate Vendure

### ISSUE THREAD

**Issue by jonyw4** [vendure-ecommerce/vendure#457]
## Problem
Today to make the Vendure localizations updated we always need to check if we are covering all `en.json` properties. For now, we have hundreds of properties, so this is an annoying job and its hard to keep track new localization properties. 

## Solution
I suggest implementing a localization service integrated with this repo. A great one is [Crowdin](https://crowdin.com/), it has a lot of features like: online translation editor, GH Integration, version management, many of file formats and the most important, it's free for open source projects.

**Comment by michaelbromley** [comment]
I'd like to try to integrate with Crowdin. I'll try to find a few hours to work on this in the near future.

**Comment by michaelbromley** [comment]
I just submitted a request for the open-source free Crowdin plan. However, one of the conditions of the plan is that:

> You do not have any commercial products related to the open-source project you are requesting a license for.

This is true of Vendure _for now_, though I _do_ have plans to offer commercial products related to the MIT parts. I made this clear in my application, so let's see what they say.

If we cannot use the free plan, I'm afraid I cannot justify $25/month for the plan we would need. In that case I'll look into some other options around l10n tooling.

**Comment by jonyw4** [comment]
You are right, but I think that applies only for the current project, in this case, this repo. If you want to make some commercial products you will need to put those in a new repository (I believe that this were already the case).
Material UI today use Crowdin in the open source repo, [and they have some commercial projects related to that](https://material-ui.com/store/).

**Comment by dlhck** [comment]
We could also have a look at https://poeditor.com/

**Comment by michaelbromley** [comment]
Closing this one after a long time — the project has since settled on lingui .po catalogs as the translation mechanism (used by the React dashboard today, with core configurable-operation strings being migrated in issue 4820), and contributions happen via PRs against those catalogs. The contributor workflow documentation is tracked in issue 3989. Adopting a separate hosted translation platform on top of that isn't planned, so closing as superseded. Thanks for raising it back in 2020!
